about/brand_use.md

```diff
@@ -0,0 +1,45 @@
+
+# Brand Use Guidelines
+
+## The purpose of these guidelines
+
+Using our logo consistently is important to present a unified message to the world about who we are. That message is: 
+arXiv is a place of connection, linking together people and ideas, and connecting them with the world of open science.
+
+Unauthorized use of our logo causes confusion about who we are. And when it’s use is authorized it’s just as important that we follow consistent guidelines to preserve the legibility, memorability, and legal defensibility that we need. Our logo, approved by Cornell University, is the legal property of arXiv and Cornell University, protected against unauthorized use. Our logo is ready to help us share a consistent, intentional, and meaningful message about who we are with the world. 
+
+### General guidelines that cover all logo use cases:
+
+1. Use of the name arXiv and associated logos, web addresses, and colors are only allowed with explicit written permission 
+from the arXiv management team, with the exception of a) arXiv authors following the guidelines for personal use below and b) current arXiv members, affiliates, and sponsors following the guidelines below.
+2. If use is allowed, display of the arXiv logo should always follow the visual layout rules as stated in the [arXiv brand guide](brand). 
+3. The logo should never be altered in any way, redrawn, used in an unspecified color, or reproduced on a background that will impair its visual recognition. 
+4. The use of the name “arXiv” or “arXiv.org,” in non-arXiv organization names or projects that imply or tend to imply some official connection with arXiv, is prohibited.
+5. Except as specifically authorized in writing, use of arXiv’s name and marks in advertising and other promotional vehicles is prohibited when such use is likely to be perceived as an endorsement, even if such an endorsement is not the intention of the person or organization seeking to use arXiv’s name or marks.
+6. In order to protect its name and reputation, arXiv regulates the use of arXiv’s name and marks on memorabilia, products, and goods. Third party use of arXiv’s logo for products and other goods is prohibited.
+
+### Further guidelines based on project type:
+
+#### If you are an author with published works on arXiv
+
+For arXiv authors only, non-commercial, personal, and minimal use of the arXiv logo for the purposes of linking from your website to arXiv’s website is allowed. In these instances the logo should follow these guidelines:
+1. The logo is displayed at a size between a minimum of 50px and a maximum of 100px in width.
+2. The logo is used with the intention of linking to arXiv (for example to your papers), and not for branding purposes.
+3. The logo must link to arXiv.
+
+Download the appropriate file for your personal use as an author: [png](https://cornell.box.com/v/arxiv-logo-small-png) or [jpg](https://cornell.box.com/v/arxiv-logo-small-jpg)
```

Comment:
Should we host these files on arXiv rather than in a Cornell box?

Reply:
Yes, in the future, that is the plan!